portfolio/src/main/webapp/script.js

```diff
@@ -49,6 +49,7 @@ function getRandomQuote() {
  * QUESTION: Why does $(document).ready() function trigger when images aren't fully loaded?
  */
 function asyncRemovePlaceHolder() {
+    console.log("remove placeholder ran");
```

Comment:
remove the console.log before submitting

Reply:
removed console.log